make/manifests.mk

```
@@ -34,7 +34,10 @@ generate-rbac: build controller-gen
 .PHONY: bundle
 bundle: clean-bundle generate-rbac kustomize ## Generate bundle manifests and metadata, then validate generated files.
 	operator-sdk generate kustomize manifests -q
-	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle --overwrite --version=${NEXT_VERSION} --channels ${CHANNEL} --default-channel ${CHANNEL} --package toolchain-member-operator
+	$(eval TMP_MANIFEST_FILE := $(shell mktemp))
+	@echo "generating manifests to temporary file: ${TMP_MANIFEST_FILE}"
+	$(KUSTOMIZE) build config/manifests -o ${TMP_MANIFEST_FILE}
+	@-cat ${TMP_MANIFEST_FILE} | operator-sdk generate bundle --overwrite --version=${NEXT_VERSION} --channels ${CHANNEL} --default-channel ${CHANNEL} --package toolchain-member-operator
```

```suggestion
	cat ${TMP_MANIFEST_FILE} | operator-sdk generate bundle --overwrite --version=${NEXT_VERSION} --channels ${CHANNEL} --default-channel ${CHANNEL} --package toolchain-member-operator
```
let's remove `@` as well as `-` so:
* it prints out the command in the output
* it ignores errors returned by the command